index.html

```diff
@@ -220,6 +220,66 @@ <h3>Multi-Post Stories</h3>
             </div>
           </section>
 
+          <!--Contact-form-->
+          <section id="contact" class="c-form">
+            <p class="con-us">CONTACT US</p>
+            <p class="con-par">
+              If you have an application you are
+              interested in developing, a feature that you
+              need built or a project that needs coding.
+              I'd love to help with it.
+            </p>
+            <form action="https://formspree.io/f/xqknvevr" method="post">
+              <ul class="un-list">
+                <li class="li-item">
+                  <label for="name">
+                    <input type="text" name="name" id="name" class="name-input detail-input" placeholder="Enter your names..."/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="name">
+                    <input type="email" name="email" id="email" class="email-input detail-input" placeholder="example@gmail.com"/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="textarea">
+                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5"></textarea>
+                  </label>
+                </li>
+                <li class="btn-container">
+                  <button type="submit" class="submit-btn">Submit</button>
+                </li>
+              </ul>
+            </form>
+
+            <hr>
+
+          </section>
```

Comment:
- [ ] Kindly make sure your design sticks to the one from form Figma as much as possible. I suggest you check every detail from the template and them on your project.

| Figma Design | Your Design |
| ------------------- | ------------------- |
| ![Screenshot from 2022-07-06 01-46-08-1](https://user-images.githubusercontent.com/83042742/177434677-90c600b0-432b-40a1-8f85-dc17eccb2e5b.png) | ![Screenshot from 2022-07-06 01-43-45](https://user-images.githubusercontent.com/83042742/177434466-f1715662-bac3-4a83-8264-a871db9852f8.png) |